### PR TITLE
Added pg_service file documentation

### PR DESCRIPTION
**Pull request by tudorbarascu**
I couldn't find at first glance any ``.. attention::``or ``.. important::`` tag, can somebody point me to it if it exists?

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/supported_data.rst [hunk 1]
```diff
@@ -479,6 +480,76 @@ To load a layer from PostGIS, perform the following steps:
...
+
+ [wastewater_service]
+ host=dbserver.com
+ dbname=water
+ user=waterpass
+
+.. note:: There are two services in the above example: ``water_service``
+  and ``wastewater_service``. You can use these to connect from QGIS,
+  pgAdmin etc. by specifying only the name of the service you want to
+  connect to (without the brackets) in their add connection tab.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: `in their add connection tab`: could you rephrase it a bit?

File: source/docs/user_manual/working_with_vector/supported_data.rst [hunk 2]
```diff
@@ -479,6 +480,76 @@ To load a layer from PostGIS, perform the following steps:
...
+  like ``export PGSERVICE=water_service``.
+
+.. note:: You can find all the parameters `here <https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS>`_
+
+.. note:: If you don't want to save the passwords in the service file you can
+  use of the `.pg_pass <https://www.postgresql.org/docs/current/static/libpq-pgpass.html>`_
+  option.
+
+
+On \*nix operating systems (GNU/Linux, macOS etc.) you can save the
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: MacOS?

**Comment by yjacolin** [comment]
Thanks, really valuable!

**Comment by yjacolin** [comment]
you can use `.. warning::` instead of `.. note::`

File: source/docs/user_manual/working_with_vector/supported_data.rst (hunk 1 above)
Comment (tudorbarascu) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: any suggestions? Will try later on when I'm fresh.

File: source/docs/user_manual/working_with_vector/supported_data.rst (hunk 2 above)
Comment (tudorbarascu) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: Starting with macOS Sierra (beta phase now).. the naming for the desktop OS will be/is **macOS**, and not Mac OS X or other variations. It's future proof.

**Comment by tudorbarascu** [comment]
Should be good now.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 3]
```diff
@@ -652,8 +654,21 @@ QGIS Server supports:
 If a short name has been set for layers, groups and project it is used by
 QGIS Sever as the layer name.
 
+Connection to service file
+--------------------------
+
+In order to make apache aware of the PostgreSQL service file (see the
+:ref:`pg-service-file` section) you need to make
+your ``*.conf`` file look like::
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Could you use :file: tag (:file:`*.conf`) for file and directory matters?

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 4]
```diff
@@ -684,7 +699,7 @@ This is a list of the variables supported by QGIS server:
   has proper permissions for writing to file. File should be created
   automatically, just send some requests to server. If it's not there, check
   permissions.
-* **QGIS_SERVER_LOG_LEVEL**: Specify desired log level. Available values are:
+* **QGIS_SERVER_LOG_LEVEL**: Specifies desired log level. See :ref:`qgis-server-logging`
 * **MAX_CACHE_LAYERS**: Specify the maximum number of cached layers (default:
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Specifies, here too and for consistency, replace any infinitive form description for other points (didn't check)

File: source/docs/user_manual/working_with_vector/supported_data.rst [hunk 5]
```diff
@@ -479,6 +480,76 @@ To load a layer from PostGIS, perform the following steps:
...
+  and ``wastewater_service``. You can use these to connect from QGIS,
+  pgAdmin etc. by specifying only the name of the service you want to
+  connect to (without the enclosing brackets).
+  If you want to use the service with ``psql`` you need to do something
+  like ``export PGSERVICE=water_service`` before doing your psql commands.
+
+.. note:: You can find all the parameters `here <https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS>`_
+
+.. note:: If you don't want to save the passwords in the service file you can
+  use of the `.pg_pass <https://www.postgresql.org/docs/current/static/libpq-pgpass.html>`_
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: use ~~of~~ the

File: source/docs/user_manual/working_with_vector/supported_data.rst [hunk 6]
```diff
@@ -479,6 +480,76 @@ To load a layer from PostGIS, perform the following steps:
...
+  option.
+
+
+On \*nix operating systems (GNU/Linux, macOS etc.) you can save the
+:file:`pg_service.conf` file in the user's home directory and
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/supported_data.rst]: .pg_service.conf?

**Comment by tudorbarascu** [comment]
Thanks for the corrections. It should be good to go now.

**Comment by DelazJ** [comment]
I'm no longer able to comment inline on my phone but there is a missing :file: line 527. Sorry.
Otherwise LGTM and thanks for this interesting section...

**Comment by tudorbarascu** [comment]
Done. Thanks for assisting @DelazJ
